pages/10.Components/03.Forms/02.Progressive Profiling/docs.en.md

```diff
@@ -17,7 +17,19 @@ There are two possible ways that you can configure a form field to display only
 
 It's recommended to use the email field in each form. From Mautic 2.9 email can be hidden, but be aware of that. Email as identifier of a contact could be unusable if same PC is used by more people (public library, schools...).
 
-1. Display field only if the value is not known yet
+1. Maximum fields displayed at a time
+
+Global feature enable progressive profiling feature and allow render only certain amount of fields in time.
+
+![image](https://user-images.githubusercontent.com/462477/80913098-3f71c500-8d42-11ea-919c-13763228bc1c.png)
+
+2. Always display
+
+Force display and skip fields orders in the form If you set **1. Maximum fields displayed at a time**. 
```

Comment:
I don't really understand this sentence. Below is my best guess at what you are trying to explain here:

```suggestion
Always force the display of the field, even if you set a maximum number of fields (x) and this field is not in the first x fields - this is helpful for fields like Email which must always be displayed.**. 
```